packages/cli/src/generator.ts

```diff
@@ -250,11 +250,9 @@ export function getNamedImportNodes(
   }
 
   const moduleSpecifier = getIdentifierName(node.moduleSpecifier.text);
-  const importIdentifier = getUniqueIdentifier(
-    typeChecker,
-    sourceFile,
-    moduleSpecifier,
-  );
+  const importIdentifier =
+    node.importClause.name?.text ??
```

Comment:
Maybe worth documenting the reason for this

Reply:
Done!